Data/DataModel/Event.cs

```diff
@@ -1,79 +1,87 @@
-﻿using System;
-using System.ComponentModel.DataAnnotations;
-using System.ComponentModel.DataAnnotations.Schema;
-
-namespace ServerCore.DataModel
-{
-    public class Event
-    {
-        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
-        public int ID { get; set; }
-
-        [Required]
-        public string Name { get; set; }
-
-        public string UrlString { get; set; }
-
-        [NotMapped]
-        public string EventID => UrlString ?? ID.ToString();
-
-        [DataType(DataType.EmailAddress)]
-        public string ContactEmail { get; set; }
-
-        public int MaxNumberOfTeams { get; set; }
-        public int MaxTeamSize { get; set; }
-        public int MaxExternalsPerTeam { get; set; }
-        public bool IsInternEvent { get; set; }
-        public DateTime TeamRegistrationBegin { get; set; }
-        public DateTime TeamRegistrationEnd { get; set; }
-
-        /// <summary>
-        /// Returns whether or not team registration is active.
-        /// </summary>
-        /// <returns>True if the current date is between the team registration begin and end times.</returns>
-        [NotMapped]
-        public bool IsTeamRegistrationActive
-        {
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace ServerCore.DataModel
+{
+    public class Event
+    {
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int ID { get; set; }
+
+        [Required]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// The URL prefix of the event, as in: www.puzzlehunt.org/urlString
+        /// </summary>
+        public string UrlString { get; set; }
+
+        [NotMapped]
+        public string EventID => UrlString ?? ID.ToString();
+
+        /// <summary>
+        /// The prefix of the partial files that provide the Home, FAQ, and Rules content.
+        /// </summary>
+        public string HomePartial { get; set; }

```

Comment:
Do you have an example of how this would look? I'm not really sure how I would reference this in an event - an example string might help (maybe as a comment?)